playground/mocks/data/idp/idx/error-okta-verify-totp.json

```diff
@@ -117,9 +117,6 @@
   "currentAuthenticator": {
     "type": "object",
     "value": {
-      "profile": {
-        "deviceName": "OnePlus 7 Pro"
-      },
```

Comment:
Do we know when this API change was made? If we can, let's reference the ticket here.

Reply:
Sure, will add that 👍 API change was just made today